src/Map.js

```diff
@@ -153,6 +153,9 @@ export default class Map {
     this.alert.mount(mountAlertTarget)
 
     this.map = this.L.map(mountTarget, mapOptions)
+    if (this.zoomControlPosition) {
+      this.map.zoomControl.setPosition(this.zoomControlPosition)
```

Comment:
when this map-core is mounted to the map-client, setPosition will throw an error for the case when the `zoomControl` is set to false in map-client and `zoomControlPosition` is passed in some truthy value. You could perhaps change the condition to `this.zoomControl && this.zoomControlPosition` on line156 to be on the safe side

![setPositionError](https://user-images.githubusercontent.com/52301822/153948307-fc2be0e2-315c-46e8-829d-c50922e478fc.png)